NotebookAnalyzers/main.py

```diff
@@ -31,6 +39,17 @@ async def nb_lint(notebook: UploadFile = File(...), bottom_size: int = Form(4)):
     os.remove(Path(config.data_path) / notebook.filename)
     return nb.get_pynblint_results()
 
+@app.post("/linters/repo-linter/")
+async def nb_lint(local_project: UploadFile = File(...), bottom_size: int = Form(4)):
```

Comment:
Why do you define this as an async function?

Reply:
In the fastAPI file upload guide it is defined like that. I did not go deeper into the matter. I did it just like I did in the end-point for the linting of the single notebook.